src/api/Base.js

```diff
@@ -123,6 +124,75 @@ class Base {
     getCache(): Cache {
         return this.cache;
     }
+
+    /**
+     * Generic success handler
+     *
+     * @param {Object} data the response data
+     * @param {Function} successCallback the success callback
+     */
+    successHandler(data: Object, successCallback: Function) {
+        if (!this.isDestroyed()) {
+            successCallback(data);
+        }
+    }
+
+    /**
+     * Generic error handler
+     *
+     * @param {Object} data the response data
+     * @param {Function} errorCallback the error callback
+     */
+    errorHandler(error: Object, errorCallback: Function) {
```

Comment:
There is an errorHandler already in one of the other files I think. Probably should consolidate.

Reply:
Looks like Item has one. Will consolidate them